src/XMakeCommandLine/UnitTests/XMake_Tests.cs

```diff
@@ -699,7 +698,22 @@ private void RobustDelete(string path)
             }
         }
 
+        /// <summary>
+        /// Run msbuild.exe within corerun.exe
+        /// It concatenates the given msbuild path to the given parameters and sends it to corerun.
+        /// </summary>
+        private string ExecMSBuild(string pathToMsBuildExe, string msbuildParameters, bool expectSuccess = true)
+        {
 #if FEATURE_RUN_EXE_IN_TESTS
```

Comment:
Hmm, this is kinda misnamed now.  @dsplaisted is there a good way to check for "if I need to wrap with corerun.exe"?